### DISCUSSION TITLE
Information for Hacktoberfest contributors!

### DISCUSSION THREAD

**Discussion by xEdziu** [xEdziu/KeepMyPassword-Desktop#88] [Announcements]
## Hi everyone!

Since it is THAT time of the year, when we celebrate all open source maintainers and contributors, I invite you all for helping improving me this project!

Currently, I am in need of translators to internationalize my application even more!

Saying so, please acknowledge yourself with those links/information:

- [Complete Guide for Translators](https://github.com/xEdziu/KeepMyPassword-Desktop/wiki/Complete-Guide-for-Translators)
- [List of already existing translations](https://github.com/xEdziu/KeepMyPassword-Desktop/tree/main/src/main/resources)
- [Issue referring to translation](https://github.com/xEdziu/KeepMyPassword-Desktop/issues/50)

I will merge all PRs after October the 1st, to make sure that you will contribute during Hacktoberfest!

Thank you all for your contributions! It means the world to me!

**Comment by Der-Zauberer** [comment]
@xEdziu  The company I work for gave us the task of participating in a random Hacktoberfest project. I would therefore like to contribute further work to this project. Are there tasks that are pending at the moment?

**Reply by xEdziu** [reply]
Hello @Der-Zauberer, currently there are no actual critical issues that can be resolved.

If you really want to improve and participate more in Hacktoberfest, you can try to make content in table cells centered, as I didn't really need to do it, but it would look much cleaner :D
![image](https://user-images.githubusercontent.com/50357817/196038872-43ac3299-eb14-46a5-b307-184bc079b2fa.png)
